api/test/h3-data/mocks/h3-alternative-fixture.ts

```diff
@@ -1,5 +1,7 @@
 export const h3AlternativeFixture = {
```

Comment:
Overall I don't like the "alternative" name used all over the place. If someone joined the project tomorrow, and saw this, they would think "ok, they are alternative, but alternative to what? Did we have some other fixtures before, or are these just hipster fixtures that listen to music that hasn't been made yet?"

Additionally, since they are hard to decode mentally, I would suggest adding a small comment on each of them, describing what they are useful for. Again, thinking about the "how could this help a dev that joins the project tomorrow" usually helps in writing a good description.

Reply:
same thing on the "expected-responses" part

Reply:
I second @tiagojsag thought on replacing the old fixtures, as they make no longer sense thanks to your brilliant contribution.

IMO the expected-responses part does no harm and it's purpose is quite understandable, but probably @tiagojsag can see further than we actually can

Reply:
Thank you for the reviews @tiagojsag and @alexeh! I think all the comments have been addressed - old fixtures replaced by the new ones, no more alternatives :) I also changed the naming a bit and added short descriptions in the fixture files.